.tfsec/_tfchecks.yml

```diff
@@ -0,0 +1,32 @@
+
+checks:
+- code: CU001
+  description: Custom check to ensure the CostCentre tag is applied to EC2 instances
+  impact: By not having CostCentre we can't keep track of billing
+  resolution: Add the CostCentre tag
```

Comment:
Did you intend to enforce "CostCentre" or is this an initial check-in?

Reply:
This is an initial checkin to show a failure mode.
Will be switched to our normal tags after that.